services/core-web/src/components/Forms/AddTailingsForm.js

```diff
@@ -31,6 +48,83 @@ export const AddTailingsForm = (props) => (
         </Form.Item>
       </Col>
     </Row>
+    <Row gutter={16}>
+      <Col md={12} xs={24}>
+        <Form.Item>
+          <Field
+            id="latitude"
+            name="latitude"
+            label="Latitude*"
+            component={RenderField}
+            validate={[number, maxLength(10), lat, required]}
+          />
+        </Form.Item>
+      </Col>
+      <Col md={12} xs={24}>
+        <Form.Item>
+          <Field
+            id="longitude"
+            name="longitude"
+            label="Longitude*"
+            component={RenderField}
+            validate={[number, maxLength(12), lon, required]}
+          />
+        </Form.Item>
+      </Col>
+    </Row>
+    <Row gutter={16}>
+      <Col md={12} xs={24}>
+        <Form.Item>
+          <Field
+            id="consequence_classification_status_code"
+            name="consequence_classification_status_code"
+            label="Consequence Classification*"
+            component={RenderSelect}
+            data={props.consequenceClassificationStatusCodeOptions}
+            validate={[required]}
+          />
+        </Form.Item>
+      </Col>
+      <Col md={12} xs={24}>
+        <Form.Item>
+          <Field
+            id="tsf_operating_status_code"
+            label="Operating Status*"
+            name="tsf_operating_status_code"
+            component={RenderSelect}
+            data={props.TSFOperatingStatusCodeOptions}
+            validate={[required]}
+          />
+        </Form.Item>
+      </Col>
+    </Row>
+    <Row gutter={16}>
+      <Col md={12} xs={24}>
+        <Form.Item>
+          <PartySelectField
+            id="eor_party_guid"
+            name="eor_party_guid"
+            label="Engineer of Record"
+            partyLabel="EoR"
+            allowNull
+            // validate={[required]}
```

Comment:
Not required?

Reply:
Correct, not required, and this has been deleted.